### PR TITLE
Added Google+ Schema To Social Links

### PR DESCRIPTION
**Pull request by ihsekat**
Structured data for social profiles: https://developers.google.com/structured-data/customize/social-profiles

### CODE REVIEW

File: website-guts/templates/layouts/wrapper.hbs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 layout_modals:
 - signin_modal_compiled
 - signup_modal_compiled
-- create_experiment_compiled
+- create_experiment_compiledf
 - reset_password_compiled
 - error_modal_compiled
```
Comment (dtothefp) [inline website-guts/templates/layouts/wrapper.hbs]: @ihsekat there is an error here in the modal names, did you accidentally change something?

Comment (ihsekat) [inline website-guts/templates/layouts/wrapper.hbs]: Ahh I see, I must have tried to type command-f and input an extra f

**Comment by ihsekat** [comment]
I fixed the typo, can merge this @dtothefp?

File: website-guts/templates/layouts/wrapper.hbs [hunk 2]
```diff
@@ -168,10 +168,11 @@ layout_modals:
                                     <li><a href="{{linkPath}}/privacy/">Privacy Policy</a> &nbsp;<span class="separater">|</span></li>
                                     <li class="security"><a href="{{linkPath}}/security/">Security</a></li>
                                 </ul>
-                                <ul class="social-links">
-                                    <li><a href="http://twitter.com/optimizely" class="twitter">Twitter</a></li>
-                                    <li><a href="http://www.facebook.com/optimizely" class="facebook">Facebook</a></li>
-                                    <li><a href="https://plus.google.com/+optimizely/posts" class="google-plus">Google+</a></li>
+                                <ul class="social-links" itemscope itemtype="http://schema.org/Organization">
+                                    <link itemprop="url" href="http://www.your-company-site.com">
+                                    <li><a itemprop="sameAs" href="http://twitter.com/optimizely" class="twitter">Twitter</a></li>
```
Comment (dtothefp) [inline website-guts/templates/layouts/wrapper.hbs]: @ihsekat this is incorrect. I believe `http://www.your-company-site.com` should be `https://www.optimizely.com/`

**Pull request merged**